imgproc.yml

```
@@ -4,13 +4,18 @@ channels:
   - nodefaults
 dependencies:
   - conda-forge::python==3.7
-  - conda-forge::matplotlib==3.5.3
+  - conda-forge::matplotlib
```

```suggestion
  - conda-forge::matplotlib==3.3.2
```
3.3.2 might work instead of 3.5.3